Thinktecture.Relay.Server/DependencyInjection/RelayServerModule.cs

```
@@ -15,14 +16,18 @@
 
 namespace Thinktecture.Relay.Server.DependencyInjection
 {
-	class RelayServerModule : Module
+	internal class RelayServerModule : Module
 	{
 		private readonly IConfiguration _configuration;
+		private readonly ICustomCodeAssemblyLoader _customCodeAssemblyLoader;
+		private readonly IControllerLoader _controllerLoader;
 		private readonly IInterceptorLoader _interceptorLoader;
 
-		public RelayServerModule(IConfiguration configuration, IInterceptorLoader interceptorLoader)
+		public RelayServerModule(IConfiguration configuration, ICustomCodeAssemblyLoader customCodeAssemblyLoader, IControllerLoader controllerLoader, IInterceptorLoader interceptorLoader)
 		{
 			_configuration = configuration ?? throw new ArgumentNullException(nameof(configuration));
+			_customCodeAssemblyLoader = customCodeAssemblyLoader ?? throw new ArgumentNullException(nameof(customCodeAssemblyLoader));
+			_controllerLoader = controllerLoader;
```

Should throw an exception when controllerLoader is null